dwave_networkx/drawing/qubit_layout.py

```diff
@@ -143,17 +125,16 @@ def node_color(v):
         # the range of the color map is shared for nodes/edges and is symmetric
         # around 0.
         vmag = max(max(abs(c) for c in node_color), max(abs(c) for c in edge_color))
-        if vmin is None:
-            vmin = -1 * vmag
-
-        if vmax is None:
-            vmax = vmag
 
-        if edge_vmin is None:
-            edge_vmin = -1 * vmag
+        # since we're applying the colormap here, matplotlib throws warnings if
+        # we provide these arguments and it doesn't use them.
+        cmap = kwargs.pop('cmap', plt.get_cmap('coolwarm'))
+        vmin = kwargs.pop('vmin', -1 * vmag)
+        vmax = kwargs.pop('vmax', vmag)
 
-        if edge_vmax is None:
-            edge_vmax = vmag
+        edge_cmap = kwargs.pop('edge_cmap', plt.get_cmap('coolwarm'))
+        edge_vmin = kwargs.pop('edge_vmin', -1 * vmag)
+        edge_vmax = kwargs.pop('edge_vmax', vmag)
```

Comment:
Actually, the same question applies to all of these.

Reply:
Fixed with https://github.com/dwavesystems/dwave-networkx/pull/233/files#diff-344ce7a800a7c461ff087cf0103c3ba81b6af1935497c0b179431a811faee391R134-R140